### PR TITLE
Remove deprecated sklearn dependency

### LINKED ISSUE #56: Rexmex uses the deprecated sklearn dependency
I recently noticed, that rexmex uses the [deprecated sklearn ](https://pypi.org/project/sklearn/) dependency. The correct scikit-learn dependency however is also present in `setup.py` and removing `sklearn` should be therefore easy and without issues.

### PR DESCRIPTION
**Pull request by dobraczka**
# Summary
 
Fixes #56 by removing deprecated sklearn dependency.
 
- [x] Code passes all tests
- [x] Unit tests provided for these changes (does not apply)
- [x] Documentation and docstrings added for these changes (does not apply)

## Changes 

* Remove deprecated sklearn dependency

### CODE REVIEW

File: docs/requirements_1.txt [hunk 1]
```diff
@@ -2,10 +2,10 @@ tqdm
...
-sklearn
+scikit-learn
 scipy
 pandas
 six
 nbsphinx
 nbsphinx_link
-jupyter-sphinx
\ No newline at end of file
+jupyter-sphinx
```
Comment (dobraczka) [inline docs/requirements_1.txt:11]: Not sure why this shows up in the diff 🤔

**Comment by kajocina** [comment]
I updated something on the main branch for the CI to stop failing. Could you pull the changes to your fork and update the PR? Otherwise it will keep on failing CI :)

**Comment by mberr** [comment]
Hi @kajocina 

do you already know when you will release this fix? We are currently seeing some [failing pipelines](https://github.com/pykeen/pykeen/actions/runs/3878460610/jobs/6614651000) because of trying to install `sklearn` fetched as recursive dependency via `rexmex`.

**Comment by kajocina** [comment]
Hey @mberr 

just released the 0.1.3 with this fix!
Thanks for reminding us.

**Comment by mberr** [comment]
Thanks for the quick response and release! 🚀

**Pull request merged**